### DISCUSSION TITLE
Friesach and cramped RAM

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#1752] [Help]
I really enjoy Friesach. But, I only have 16 GiB of RAM, and that is the motherboard max. So I cannot load the Solowerk :-(
Any ideas of how to solve this, except for new motherboard  and more RAM ;-) Is it pooible to compress the wav files or to reduce the quality (my loadspeakers are not that great).

(Ubuntu 22.04 LTS, GO 3.12.1-1)

//Jens

**Comment by larspalo** [comment]
Usually, the recommendation is to load as 20 bit and apply lossless compression - it's possible that it would be enough to load properly. This makes the interpolation method to fall back to linear, but it's a price to pay. If the sample set is still too large - load as 16 bit and apply lossless compression.

Next steps would be to set attacks/releases to load only one single, and if still too large - load only first loop.

Personally I don't go lower than 16 bit sample loading and the attack/release/loop reductions. If a sample set is still too large, I just remove what I don't deem necessary (surround rear channels usually go first) until I get an usable organ.

**Reply by larspalo** [reply]
I'm not sure what Friesach version you're using, but the only GrandOrgue version (not full image resolution) that I have will load with a RAM consumption of less than 3.4 GB when I set the sample loading options to:

Lossless compression (checked)
Stereo
16 bit
First Loop
Single attack
Single release

Thus, it should be possible to use more of the features/better sample loading settings without an issue if one had 16 GB of RAM.

**Reply by jason-barba** [reply]
Thanks Lars for quick answer. It showed I had already set 16 bits. Adding Lossless compression and Stereo reduced the Friesach RAM foot print:

16 bit, stereo, no compression
----------------------
jens@orgel:~$ free -m
               total        used        free      shared  buff/cache   available
Mem:           15880        1003         557        9820       14319        4740
Swap:           2047           3        2044



16 bit, stereo, losssless compression
-----------------------------
jens@orgel:~$ free -m
               total        used        free      shared  buff/cache   available
Mem:           15880         965        5691        5454        9223        9139
Swap:           2047          84        1963

These settings I applied for th Friesach sampleset. But I found the same options in the overall GO settings pane (File/Settings). When is it adviced to use the general settings and when the per sampleset settings?

I will return with the outcome of loading also the Solowerk ;-)

God Jul!

**Reply by ClarkeBrunt** [reply]
I don't know why the difference, but as I wrote early in 2023 on another thread #1407 "_The Friesach organ fits in 16GB RAM without missing anything out, or reducing the sample size below 24 bits. It uses maybe 12-13GB - this is with lossless compression enabled (and on Windows 10, should it matter)."_ So it fits for me, at full bit-depth and without missing anything out. I've since switched to running on Linux Ubuntu Studio on the same 16GB computer - it still fits. I haven't knowingly altered other settings to make it fit.

**Reply by jason-barba** [reply]
Success! Even the Solowerk loads. I suggest it is the lossless compression that makes it possible rather that the 16 bits.

**Reply by larspalo** [reply]
@jason-barba Yes, the lossless compression should generally be the first approach and the loss of going to 20 bit is the next if you really need to lower RAM usage.

God Jul!

**Reply by jason-barba** [reply]
@ClarkeBrunt Sorry I missed your earlier thread. But we have the same experience: the compression is the thing!

**Reply by Aduragbemi-Adeyinka** [reply]
I fit Friesach into 8GB RAM; applying loss less compression, loading single attack and single release, not loading any noise effect and some stops eg 32'stops on pedals and 16'stops on manuals. I think I left the resolution at 24bit but I'm not sure.
